spec/lib/query/proposal/search_spec.rb

```diff
@@ -1,42 +1,66 @@
+require 'ansi/code'
+
 describe Query::Proposal::Search do
+
+  before do
+    refresh_index
```

Comment:
why do we need to refresh the index if we are calling `reindex` in almost every test? do we need to refresh before we reindex?

Also, care to inline this  where needed rather than putting in a `before` per the general spec refactor effort?